dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/DatabaseContainerUtil.java

```diff
@@ -0,0 +1,138 @@
+package com.ibm.ws.concurrent.persistent.fat.multiple;
+
+import java.io.File;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.ibm.websphere.simplicity.config.DataSource;
+import com.ibm.websphere.simplicity.config.DataSourceProperties;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.dsprops.Properties;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_db2_jcc;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_informix;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_microsoft_sqlserver;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_oracle;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_sybase;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyServer;
+
+public class DatabaseContainerUtil {
+	//Logging Constants
+	private static final Class<DatabaseContainerUtil> c = DatabaseContainerUtil.class;
+	private static final String nl = System.getProperty("line.separator");
+	
+    /**
+     * Gets DataSource connection properties from JdbcDatabaseContainer, such as, username, password, and url.
+     * Then adds these properties to the server's environment variables.
+     * And, optionally, updates server config properties dependent on the type of database being used.
+     * 
+     * @param serv - LibertyServer server instance being used for this FAT suite.
+     * @param cont - JdbcDatabaseContainer instance being used for database connectivity.
+     * @param type - DBContainerType type information about this container.
+     */
+    public static void setupDataSourceProperties(LibertyServer serv, JdbcDatabaseContainer<?> cont, DatabaseContainerType type) throws Exception {
```

Comment:
I would prefer we didn't have this helper method, and instead just let the FAT's `@BeforeClass` set properties according to the env vars it has chosen. For example:
```java
server.addEnvVar("DB_DRIVER", type.getDriverName());
server.addEnvVar("DB_HOST", cont.getContainerIpAddress());
server.addEnvVar("DB_PORT", cont.getFirstMappedPort());
// etc...
```
this will be helpful because the FAT developer can declare whatever env vars they want, instead of needing to choose the specific variables names such as `DB_DRIVER` in their server.xml config.